Test/GherkinFeatureFiles/ui-interaction.feature

```diff
@@ -0,0 +1,52 @@
+Feature: UI Interaction for Point Details
+
+  This feature verifies the interaction with a data point on the graph will display detail of the point
+
+  Background:
+    Given the user has Oculus Quest 2
+    And the user has loaded the Quest Browser
+    And the user has loaded the WebXR application
+    And the user has loaded a csv
+    And the data points are being displayed on the graph
+
+  @manualtesting
+  Scenario Outline: Clicking a data point on the graph
+    Given data points show on the graph from the csv
+    When the user selects a single data point
+    Then details of that point should appear <datapoint1> <Name1> <Province1> <Country1>
+
+    Examples:
+      | datapoint1 | Name1    | Province1 | Country1 |
+      | p1         | John Doe | Alberta   | Canada   |
+
+  @manualtesting
+  Scenario Outline: Clicking a second data point on the graph
```

Comment:
Is there any way to test for breaking things? what happens when we click and there isn't anything to interact with? does it throw any errors

Reply:
The only interactable thing is the data points. So if there are none then nothing will happen.